make/photon/db/docker-entrypoint.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 set -e
 
+PG_VERSION_NEW=$1
+
+PGBINOLD="/usr/local/pg96/bin/"
+PGBINNEW="/usr/bin"
+PGDATAOLD="/tmp/pg/v9.6"
+PGDATANEW="/tmp/pg/v13"
```

Comment:
Copy database files from volume mounted from the host directory to the directory of the container may take the disk during the container is running even the files in `/tmp/pg` are deleted after the upgrade.